src/main.cpp

```diff
@@ -214,28 +230,31 @@ MAKE_HOOK_MATCH(LevelPlay, &SinglePlayerLevelSelectionFlowCoordinator::StartLeve
     levelStarted();
 }
 
-void replayPostCallback(ReplayUploadStatus status, string description) {
+void replayPostCallback(ReplayUploadStatus status, string description, float progress) {
     if (status == ReplayUploadStatus::finished) {
         PlayerController::Refresh();
     }
-    QuestUI::MainThreadScheduler::Schedule([status, description] {
+
+    QuestUI::MainThreadScheduler::Schedule([status, description, progress] {
         uploadStatus->SetText(description);
         switch (status)
         {
-        case ReplayUploadStatus::finished:
-            leaderboardViewController->Refresh(true, true);
-            break;
-        case ReplayUploadStatus::error:
-            retryButton->get_gameObject()->SetActive(true);
-            break;
-        case ReplayUploadStatus::inProgress:
-            break;
+            case ReplayUploadStatus::finished:
+                leaderboardViewController->Refresh(true, true);
+                break;
+            case ReplayUploadStatus::error:
+                retryButton->get_gameObject()->SetActive(true);
+                break;
+            case ReplayUploadStatus::inProgress:
+                if(progress >= 100)
+                    uploadStatus->SetText("<color=#b103fcff>Posting replay: Finishing up...");
+                break;
         }
     });
 }
 
 void processResults(SinglePlayerLevelSelectionFlowCoordinator* self, LevelCompletionResults* levelCompletionResults, IDifficultyBeatmap* difficultyBeatmap, bool practice) {
-    replay->info->score = levelCompletionResults->rawScore;
+    replay->info->score = levelCompletionResults->modifiedScore;
```

Comment:
This is not the same score. Everything else expects info->score to be the base score

Reply:
Modified is the score after the multiplier has been applied (i dug through the game code to see what is used for the score in the base game)
![image](https://user-images.githubusercontent.com/55860414/159104778-c1151a72-56b0-4f1a-ae3e-8a66add19883.png)

Reply:
it is correct, you can look at my scores from the past week

Reply:
No, looked at my yesterday scores and it's reapplying modifiers
<img width="354" alt="Screenshot 2022-03-19 at 09 47 44" src="https://user-images.githubusercontent.com/8793817/159116268-75517a91-4eec-451f-b285-c71b0c0d721d.png">